docs/develop/develop-add-ins-vscode.md

```diff
@@ -25,6 +25,12 @@ If you're using VS Code as your integrated development environment (IDE), you sh
 
 To create an Office Add-in with the Yeoman generator, follow instructions in the [5-minute quick start](../index.yml) that corresponds to the type of add-in you'd like to create.
 
+## Create the add-in project using the Office Add-ins Development Kit (preview)
+
+The [Office Add-ins Development Kit](https://marketplace.visualstudio.com/items?itemName=msoffice.microsoft-office-add-in-debugger) is a Visual Studio Code extension that allows you to create new projects directly from VS Code. For information on installing the extension and creating projects from templates and samples, see [Create Office Add-in projects using Office Add-ins Development Kit for Visual Studio Code](development-kit-overview.md).
+
+[!INCLUDE [Information about the preview status of the dev kit.](../includes/dev-kit-preview-note.md)]
+
 ## Develop the add-in using VS Code
```

Comment:
Should we combine this section together with line22~26? As they both talk about yeoman.

Reply:
Does it? I don't see the yeoman reference in this Dev Kit section.

Reply:
I mean the "Develop the add-in using VS Code" section is also about Yeoman. It seems that the Dev Kit section (line 28-32) is in between 2 Yeoman sections (line22-26 and line34-40).